appconfiguration/client/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java

```diff
@@ -40,24 +59,52 @@
  *     .build();
  * </pre>
  *
+ * @see ConfigurationAsyncClient
  * @see ConfigurationClient
  * @see ConfigurationClientCredentials
  */
 public final class ConfigurationClientBuilder {
-    private final ConfigurationAsyncClientBuilder builder;
+    // This header tells the server to return the request id in the HTTP response. Useful for correlation with what
+    // request was sent.
+    private static final String ECHO_REQUEST_ID_HEADER = "x-ms-return-client-request-id";
+    private static final String CONTENT_TYPE_HEADER = "Content-Type";
+    private static final String CONTENT_TYPE_HEADER_VALUE = "application/json";
+    private static final String ACCEPT_HEADER = "Accept";
+    private static final String ACCEPT_HEADER_VALUE = "application/vnd.microsoft.azconfig.kv+json";
 
-    ConfigurationClientBuilder() {
-        builder = ConfigurationAsyncClient.builder();
-    }
+    private final List<HttpPipelinePolicy> policies;
+    private final HttpHeaders headers;
+
+    private ConfigurationClientCredentials credentials;
+    private URL serviceEndpoint;
+    private HttpClient httpClient;
+    private HttpLogDetailLevel httpLogDetailLevel;
+    private HttpPipeline pipeline;
+    private RetryPolicy retryPolicy;
+    private Configuration configuration;
 
+    /**
+     * The constructor with defaults.
+     */
+    public ConfigurationClientBuilder() {
+        retryPolicy = new RetryPolicy();
+        httpLogDetailLevel = HttpLogDetailLevel.NONE;
+        policies = new ArrayList<>();
+
+        headers = new HttpHeaders()
+            .put(ECHO_REQUEST_ID_HEADER, "true")
+            .put(CONTENT_TYPE_HEADER, CONTENT_TYPE_HEADER_VALUE)
+            .put(ACCEPT_HEADER, ACCEPT_HEADER_VALUE);
+    }
     /**
      * Creates a {@link ConfigurationClient} based on options set in the Builder. Every time {@code build()} is
-     * called, a new instance of {@link ConfigurationClient} is created.
+     * called, a new instance of {@link ConfigurationAsyncClient} is created.
```

Comment:
It is not an async client that is being built - this method builds a sync client.